java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S5804_java.html

```diff
@@ -37,6 +37,19 @@ <h2>Sensitive Code Example</h2>
   // ....
 }
 </pre>
+<ul>
+  <li> <a
+  href="https://docs.spring.io/spring-security/site/docs/3.0.x/apidocs/org/springframework/security/core/userdetails/UsernameNotFoundException.html">UsernameNotFoundException</a> is thrown (except when it is in the loadUserByUsername method): </li>
+</ul>
+<pre>
+public String authenticate(String username, String password) {
+  // ....
+  if(user == null) {
+      throw new UsernameNotFoundException("user not found"); // Sensitive
+  }
+  // ....
+}
+</pre>
```

Comment:
@margarita-nedzelska-sonarsource Can you confirm that this description update matches the implementation of the rule?

Reply:
Yes, LGTM!